src/Compile.hs

```diff
@@ -378,17 +392,17 @@ instance Compilable AST where
   compile rootPath outputPath adt =
     let
       exps         = aexps adt
-      adts         = aadts adt
+      typeDecls    = atypedecls adt
       path         = apath adt
       imports      = aimports adt
 
 
       path'        = fromMaybe "Unknown" path
 
       infoComment  = "// file: " <> path' <> "\n"
-      helpers      = curryPowder
+      helpers      = curryPowder <> "\n" <> eq
```

Comment:
Very minor, but we might want to globally rename `curryPowder` to something like `autocurry`. Probably outside the scope of this PR, relative to size.

Reply:
Linked this to the ticket: https://github.com/open-sorcerers/madlib/projects/1#card-50063255